addons/sale_stock/tests/test_sale_stock.py

```diff
@@ -1033,3 +1033,32 @@ def test_reservation_method_w_sale(self):
         sale_order4.action_confirm()
         self.assertTrue(sale_order4.picking_ids)
         self.assertEqual(sale_order4.picking_ids.state, 'assigned')
+
+    def test_15_cancel_delivery(self):
+        """
+        Suppose the option "Lock Confirmed Sales" enabled and a product with the invoicing policy set to "Delivered
+        quantities". When cancelling the delivery of such a product, the invoice status of the associated SO should be
+        'Nothing to Invoice'
+        """
```

Comment:
Do you mind if the description is formatted like that ?
```suggestion
        """ Suppose the option "Lock Confirmed Sales" enabled and a product with the invoicing
        policy set to "Delivered quantities". When cancelling the delivery of such a product, the
        invoice status of the associated SO should be 'Nothing to Invoice'
        """
```
As we don't put a break line after the first """ usually, and to split the description evenly on the three lines (so we don't pass the soft limit of 100 chars by line).